plugins/destination/csv/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/destination/csv/client"
+	"github.com/cloudquery/cloudquery/plugins/destination/csv/resources/plugin"
+	"github.com/cloudquery/plugin-sdk/plugins"
+	"github.com/cloudquery/plugin-sdk/serve"
+)
+
+const (
+	// TODO: add sentry DSN
+	sentryDSN = ""
```

Comment:
Let's remember to do this before merging ☝️

Reply:
Done